package.json

```diff
@@ -23,10 +23,10 @@
   },
   "dependencies": {
     "hapi": "^16.3.0",
-    "hapi-confi": "^4.0.0",
+    "hapi-confi": "^4.0.1",
```

Comment:
was hapi-confi released as patch?  seems like we added a new feature, so should have been minor